src/styles/components/_code-block.scss

```diff
@@ -0,0 +1,56 @@
+$breakpoint-tablet: 820px;
+
+.cmp-code-block {
+  width: calc(100% - 2.5rem); // width 100% - 1.25rem on either side; 20px/16
+  margin: auto;
+  max-width: 100rem; // 1600px/16
+
+  @media only screen and (min-width: $breakpoint-tablet) {
+    width: 100%;
+    padding: 3.75rem 8.5625rem; // top&bottom: 60px/16; left&right: 137px/16
+  }
+
+  &__heading {
+    font-size: 1.5625rem; // 25px/16
+    font-family: Viga, sans-serif;
+
+    &--arrow {
+      transform: rotate(-30deg);
+      max-width: 13%; // visually this looked correct
+      position: relative;
+      top: 2.5rem; // 24px/16
+      left: 1rem; // 16px/16
+    }
+  }
+
+  &__content {
+    padding: 1.875rem 1.25rem; // top&bottom: 30px/16; left&right: 20px/16
+    background-color: map.get($colors, "black");
+    color: map.get($colors, "white");
+    line-height: 1.75rem; // 28px/16
+    font-size: map.get($font-sizes, "xs");
+    font-family: Oxygen, sans-serif;
+    letter-spacing: 2%;
+    margin-top: 3rem;
+
+    @media only screen and (min-width: $breakpoint-tablet) {
+      background-color: map.get($colors, "white");
+      color: map.get($colors, "black");
+      border: 2px solid map.get($colors, "black");
+      padding: 2.5rem 3.75rem; // top&bottom: 40px/16; left&right: 60px/16
+    }
+  }
+
+  pre {
+    margin: 0;
+    white-space: pre-wrap;
+    word-break: break-all;
+    overflow-wrap: break-word;
```

Comment:
Is there any better way to specify how the content with the `code` tags breaks, specifically for mobile? It's just a little hard to read.

Reply:
There's a bit of a trade-off for whether we want the code to wrap or to have horizontal scrolling. It's fairly typical to use `white-space: pre` and `overflow-x: scroll` (or `overflow-x: auto`). [css-tricks](https://css-tricks.com) does this, for example. If we went that route, we would not want to have vertical scrolling, since it's bad practice to have scrolling in two dimensions for the same element. We would be able to remove the `word-break` and `overflow-wrap` properties here, though.

Reply:
Thank you for this! Knowing that we probably _do_ want this to scroll made that a lot easier. I couldn't find the right combination of the properties and that `overflow-x` with `white-space:pre` made all the difference.